### PR TITLE
allow for variable partitioning of ssw in accessom coupler

### PR DESCRIPTION
**Pull request by russfiedler**

### CODE REVIEW

File: src/accessom_coupler/mom_oasis3_interface.F90 [hunk 1]
```diff
@@ -71,6 +71,18 @@ module mom_oasis3_interface_mod
 !   <DATA NAME="send_after_ocean_update" TYPE="logical" DEFAULT=".FALSE.">
 !    TRUE if coupling strategy requires we send data to coupler AFTER updating the ocean
 !   </DATA>
+!   <DATA NAME=""frac_vis_dir TYPE="real" DEFAULT="0.215">
```
Comment (aekiss) [inline src/accessom_coupler/mom_oasis3_interface.F90:74]: should be
<DATA NAME="frac_vis_dir" TYPE="real" DEFAULT="0.215">

File: src/accessom_coupler/mom_oasis3_interface.F90 [hunk 2]
```diff
@@ -71,6 +71,18 @@ module mom_oasis3_interface_mod
...
+!    Fraction of ssw apportion to direct visible radiation
+!   </DATA>
+!   <DATA NAME=""frac_vis_dif TYPE="real" DEFAULT="0.215">
```
Comment (aekiss) [inline src/accessom_coupler/mom_oasis3_interface.F90:77]: as above

File: src/accessom_coupler/mom_oasis3_interface.F90 [hunk 3]
```diff
@@ -71,6 +71,18 @@ module mom_oasis3_interface_mod
...
+!    Fraction of ssw apportioned to diffuse visible radiation
+!   </DATA>
+!   <DATA NAME=""frac_nir_dir TYPE="real" DEFAULT="0.285">
```
Comment (aekiss) [inline src/accessom_coupler/mom_oasis3_interface.F90:80]: ditto

File: src/accessom_coupler/mom_oasis3_interface.F90 [hunk 4]
```diff
@@ -71,6 +71,18 @@ module mom_oasis3_interface_mod
...
+!    Fraction of ssw apportioned to direct near infrared radiation
+!   </DATA>
+!   <DATA NAME=""frac_nir_dif TYPE="real" DEFAULT="0.285">
```
Comment (aekiss) [inline src/accessom_coupler/mom_oasis3_interface.F90:83]: ditto

File: src/accessom_coupler/mom_oasis3_interface.F90 [hunk 5]
```diff
@@ -71,6 +71,18 @@ module mom_oasis3_interface_mod
 !   <DATA NAME="send_after_ocean_update" TYPE="logical" DEFAULT=".FALSE.">
 !    TRUE if coupling strategy requires we send data to coupler AFTER updating the ocean
 !   </DATA>
+!   <DATA NAME="frac_vis_dir TYPE="real" DEFAULT="0.215">
```
Comment (aekiss) [inline src/accessom_coupler/mom_oasis3_interface.F90:74]: you're still missing the closing " for these 4 NAMES

Comment (russfiedler) [reply src/accessom_coupler/mom_oasis3_interface.F90]: Aha, it was a misplaced quote, not a doubling up.

**Comment by aekiss** [comment]
OK all looks good to me now, thanks @russfiedler

**Pull request merged**